$MirOS: ports/lang/egcs/gcc4.4/patches/patch-gcc_doc_cpp_1,v 1.1 2009/12/06 15:52:18 tg Exp $
--- gcc/doc/cpp.1.orig	Tue Mar 13 09:47:07 2012
+++ gcc/doc/cpp.1	Sun Dec  1 04:09:32 2013
@@ -123,17 +123,17 @@
 .rm #[ #] #H #V #F C
 .\" ========================================================================
 .\"
-.IX Title "CPP 1"
-.TH CPP 1 "2012-03-13" "gcc-4.4.7" "GNU"
+.IX Title "CPP-VSS 1"
+.TH CPP\-VSS 1 "2012-03-13" "gcc-4.4.7" "GNU"
 .\" For nroff, turn off justification.  Always turn off hyphenation; it makes
 .\" way too many mistakes in technical documents.
 .if n .ad l
 .nh
 .SH "NAME"
-cpp \- The C Preprocessor
+cpp\-VSS \- The C Preprocessor
 .SH "SYNOPSIS"
 .IX Header "SYNOPSIS"
-cpp [\fB\-D\fR\fImacro\fR[=\fIdefn\fR]...] [\fB\-U\fR\fImacro\fR]
+cpp\-VSS [\fB\-D\fR\fImacro\fR[=\fIdefn\fR]...] [\fB\-U\fR\fImacro\fR]
     [\fB\-I\fR\fIdir\fR...] [\fB\-iquote\fR\fIdir\fR...]
     [\fB\-W\fR\fIwarn\fR...]
     [\fB\-M\fR|\fB\-MM\fR] [\fB\-MG\fR] [\fB\-MF\fR \fIfilename\fR]
